contribute.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Standup",
+    "description": "Web app that logs daily standup updates",
+    "repository": {
+        "url": "https://github.com/mozilla/standup",
+        "license": "BSD-3-Clause",
+        "tests": "https://travis-ci.org/mozilla/standup/"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/Standup",
```

Comment:
I think I'm inclined to "land this now and write up an issue and not do anything with the issue until someone chimes in with a real-world problem".